tests/providers/amazon/aws/sensors/test_emr_step.py

```diff
@@ -150,7 +154,7 @@ def setUp(self):
             step_id='s-VK57YR1Z9Z5N',
             aws_conn_id='aws_default',
         )
-
+        # raise Exception(self.step_id)
```

Comment:
is this intentional?

Reply:
Hi Kaxil, that was just for testing purposes... realize i never removed that. Should i remove and push? Kind regards

Reply:
Yes please remove and push the commit

Reply:
Removed and pushed. Thanks @kaxil